scripts/api/data/licenses/licenseODC-By-1.0.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "ODC-By-1.0",
+  "uri": "https://opendatacommons.org/licenses/by/1.0/",
```

Comment:
This seems to be the right uri. There's only one at https://github.com/datacite/bracco/blob/df4ac2030774324adeaa72e2978c29dfb463ee3d/app/spdx.js#L3018